source/fundamentals/crud/count.txt

```diff
@@ -1,16 +1,40 @@
 .. uses tutorial.rst
 
-If we just want to know how many documents match a query we can
-perform a the ``~pymongo.collection.Collection.count_documents`` method operation
-instead of a full query. We can get a count of all of the documents
-in a collection:
+.. _pymongo-count:
+
+=====
+Count
+=====
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
```

Comment:
```suggestion

.. meta::
   :keywords: number
   
```

or whatever you think makes sense here

Reply:
Thanks, I couldn't think of anything for this page so that works